selector/__init__.py

```diff
@@ -383,33 +383,34 @@ def create_pool(self) -> Dict[Site, List[Visit]]:
         scheduling_groups = self.collector.scheduling_groups
         visits = {site: [] for site in self.sites}
 
+        print(self.sites)
```

Comment:
This should probably be deleted, or at least changed to a `logging.info` statement.